tests/microbenchmarks.py

```diff
@@ -47,11 +52,14 @@ def test_booleans(test_case):
 
             x = True and (not False) and True and (not False) and False
             y = (False != False) != False
-        """), timed=True)
+        """),
+        timed=True)
+
 
 def test_global_var_load(test_case):
     print("Running", "test_global_var_load")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
```

Comment:
At column 9: (F405) 'adjust' may be undefined, or defined from star imports: utils